src/conventional/metar_csv2ioda.py.in

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2020 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# author: Greg Thompson gthompsn AT ucar DOT edu
+# This script will work with decoded, CSV files created by gthompsn stored at NCAR
+#   data-access.ucar.edu:/glade/campaign/ral/aap/gthompsn/METARs/2019/20191231/2019123118_metars.csv.gz
+#
+
+from __future__ import print_function
```

Comment:
You should not need this now that the interpreter is explicitly declared as python3. The print syntax changed from python2 to python3 and this statement was a workaround to allow python2 print statements to use the python3 syntax.

Reply:
OK, got it. I removed this line.